mitiq/shadows/classical_postprocessing.py

```diff
@@ -368,6 +368,7 @@ def expectation_estimation_shadow(
                 f_val = f_est.get(b, None)
                 if f_val is None:
                     means.append(0.0)
+                    continue
```

Comment:
```suggestion
                    product = 0.0
```
Is this equivalent? If so, I would suggest using `product = 0.0` since it makes the workflow more linear and easier to debug in the future. 

If my suggestion doesn't work, I am fine with `continue`.

Reply:
That makes sense. Should a similar thing be done with the "else" on line 374 (when there aren't any matches between the measurement_outcome and the pauli_str expected outcomes.) That else would also include a product = 0.0 and at the end of the for loop do a single means.append(np.sum(product) / len(idxes))?